modules/binding-infra-node-workloads-using-taints-tolerations.adoc

```diff
@@ -151,4 +113,6 @@ This toleration matches the taint created by the `oc adm taint` command. A pod w
 Moving pods for an Operator installed via OLM to an infra node is not always possible. The capability to move Operator pods depends on the configuration of each Operator.
 ====
 
-. Schedule the pod to the infra node using a scheduler. See the documentation for _Controlling pod placement onto nodes_ for details.
+. Schedule the pod to the infra node using a scheduler. See the documentation for "Controlling pod placement using the scheduler" for details.
```

Comment:
```suggestion
. Schedule the pod to the infra node by using a scheduler. For more information, see "Controlling pod placement using the scheduler".
```
- https://www.ibm.com/docs/en/ibm-style?topic=word-usage#using
- https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentences